src/fides/api/main.py

```diff
@@ -248,8 +250,9 @@ def sanitise_url_path(path: str) -> str:
     """Returns a URL path that does not contain any ../ or //"""
     path = unquote(path)
     path = os.path.normpath(path)
+
     for token in path.split("/"):
-        if ".." in token:
+        if ".." in token and not re.search(NEXT_JS_CATCH_ALL_SEGMENTS_RE, token):
```

Comment:
won't this still allow the `example.com/[datasetName]/[collectionName]/[...subFields]-js/../../../../etc.` scenario?

Reply:
I don't think so, because of the `.split("/")` above. So it's testing each path segment like:
* token = `example.com`
* token = `[datasetName]`
* token = `[collectionName]`
* token = `[...subFields]-js`
* token = `..`
* token = `..`
* token = `..`
* token = `..`

Reply:
Yeah, makes sense! Thanks for the illustration :)